### PR TITLE
Support CrowdPose dataset

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

File: configs/bottom_up/higherhrnet/README.md [hunk 1]
```diff
@@ -21,10 +21,17 @@
 | [HigherHRNet-w32](/configs/bottom_up/higherhrnet/coco/higher_hrnet32_coco_640x640.py)  | 640x640 | 0.686 | 0.871 | 0.747 | 0.733 | 0.898 | [ckpt](https://download.openmmlab.com/mmpose/bottom_up/higher_hrnet32_coco_640x640-a22fe938_20200712.pth) | [log](https://download.openmmlab.com/mmpose/bottom_up/higher_hrnet32_coco_640x640_20200712.log.json) |
 | [HigherHRNet-w48](/configs/bottom_up/higherhrnet/coco/higher_hrnet48_coco_512x512.py)  | 512x512 | 0.686 | 0.873 | 0.741 | 0.731 | 0.892 | [ckpt](https://download.openmmlab.com/mmpose/bottom_up/higher_hrnet48_coco_512x512-60fedcbc_20200712.pth) | [log](https://download.openmmlab.com/mmpose/bottom_up/higher_hrnet48_coco_512x512_20200712.log.json) |
 
-### Results on COCO val2017 with multi-scale test
+### Results on COCO val2017 with multi-scale test with scales [2, 1, 0.5].
```
Comment (innerlee) [inline configs/bottom_up/higherhrnet/README.md:24]: gramma issue

File: configs/bottom_up/higherhrnet/crowdpose/higher_hrnet32_crowdpose_512x512.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    dataset_channel=channel_cfg['dataset_channel'],
+    inference_channel=channel_cfg['inference_channel'],
+    num_scales=2,
+    scale_aware_sigma=False,
+)
+
+# model settings
+model = dict(
+    type='BottomUp',
+    pretrained='models/pytorch/imagenet/hrnet_w32-36af842e.pth',
```
Comment (innerlee) [inline configs/bottom_up/higherhrnet/crowdpose/higher_hrnet32_crowdpose_512x512.py:52]: what if we use url in pretrained? i guess mmcv will automatically download the model from web.

Comment (jin-s13) [reply configs/bottom_up/higherhrnet/crowdpose/higher_hrnet32_crowdpose_512x512.py:52]: Okay. I will replace them with urls in a separate pr.

File: mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py [hunk 3]
```diff
@@ -53,12 +53,12 @@ def __init__(self,
         self.ann_info['inference_channel'] = data_cfg['inference_channel']
         self.ann_info['dataset_channel'] = data_cfg['dataset_channel']
 
-        self.db = []
+        self.image_set_index = []
```
Comment (innerlee) [inline mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py:56]: how to understand `image set index`? `image set`'s index? image `set index`? what is an `index`?
why is this name better than `db`?

**Comment by innerlee** [comment]
looks good

File: mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py (hunk 3 above)
Comment (jin-s13) [reply mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py:56]: change the name to `image_index`

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
need a follow up pr to update repo's readme and changelog

**Pull request merged**